### ISSUE TITLE
Removed lineage designations

### ISSUE THREAD

**Issue by erikwolfsohn** [cov-lineages/lineages-website#25]
It seems like a number of lineages were removed in b0bfdd992ca5fcd4d783b8a0bedec57b517ab498

I was wondering if there is any information about why they were removed/how to redesignate specimens assigned to those lineages.

I've attached a list of the removed files. Appreciate the help!

[missing_lineages.txt](https://github.com/cov-lineages/lineages-website/files/10015625/missing_lineages.txt)

**Comment by rmcolq** [comment]
As raised in Issue #21, the`_data/lineage_data.json` file had become too large and was therefore preventing the website from updating. We tested out using git LFS instead, but that does not integrate with github pages which render the website. As a result, we have restricted the website to include only those lineages seen in the last year. Missing lineages remain valid lineages, and due to the way this is automated, if a dormant lineage is seen again it will be added back to the page. An unfiltered JSON file can also be found  in `_data/lineage_data.full.json` for reference - this is stored with git LFS so is not used by website.
